frontend/src/bundles/studio/store/slice.ts

```diff
@@ -80,11 +85,18 @@ const { reducer, actions, name } = createSlice({
         addScript(state, action: PayloadAction<string>) {
             const script = {
                 id: uuidv4(),
-                duration: MIN_SCRIPT_DURATION,
+                duration: MIN_SCRIPT_DURATION, // You could pass duration if needed
```

Comment:
When script is created it does not have audio, so we don't know its duration. This is the reason we create script with default duration. The duration is set only after text-to-speech generation when we receive audioUrl.
So, please, remove this comment